src/phoenix/experimental/evals/functions/classify.py

```diff
@@ -112,8 +109,8 @@ def llm_classify(
         if not use_openai_function_call:
             raw_string = response
             if provide_explanation:
-                # TODO: support explanation without function calling
-                explanations.append(None)
+                raw_string, explanation = _search_for_label(raw_string), raw_string
```

Comment:
clarification: I'd make the code here clear that it's no longer a raw_string, that it is in fact a label now